apps/src/templates/sectionProgress/detail/VirtualizedDetailView.jsx

```diff
@@ -109,11 +125,11 @@ class VirtualizedDetailView extends Component {
     this.detailView = ref;
   }
 
-  onClickLevel = lessonOfInterest => {
+  onClickLevel(lessonOfInterest) {
```

Comment:
Out of curiosity, why did you move away from arrow functions and move to binding instead?

Reply:
so glad you asked! https://medium.com/@charpeni/arrow-functions-in-class-properties-might-not-be-as-great-as-we-think-3b3551c440b1

Reply:
FYI, see [here](https://github.com/code-dot-org/code-dot-org/pull/38772#discussion_r565761973). you're welcome to get in on the best-practices fun with Jessie, Maddie, and I once i circle back to it if you're interested!

Reply:
Yeah, I'd love to!